salt/states/netconfig.py

```diff
@@ -99,6 +99,358 @@ def _update_config(template_name,
+
+        update_policy_name:
+          netconfig.replace_pattern:
+            - pattern: OLD-POLICY-NAME
+            - repl: new-policy-name
+            - debug: true
+    '''
+    ret = salt.utils.napalm.default_ret(name)
+    # the user can override the flags the equivalent CLI args
+    # which have higher precedence
+    test = __salt__['config.merge']('test', test)
+    debug = __salt__['config.merge']('debug', debug)
+    commit = __salt__['config.merge']('commit', commit)
+    replace = __salt__['config.merge']('replace', replace)  # this might be a bit risky
+    replace_ret = __salt__['net.replace_pattern'](pattern,
+                                                  repl,
+                                                  count=count,
+                                                  flags=flags,
+                                                  bufsize=bufsize,
+                                                  append_if_not_found=append_if_not_found,
+                                                  prepend_if_not_found=prepend_if_not_found,
+                                                  not_found_content=not_found_content,
+                                                  search_only=search_only,
+                                                  show_changes=show_changes,
+                                                  backslash_literal=backslash_literal,
+                                                  source=source,
+                                                  path=path,
+                                                  test=test,
+                                                  replace=replace,
+                                                  debug=debug,
+                                                  commit=commit)
+    return salt.utils.napalm.loaded_ret(ret, replace_ret, test, debug)
+
+
+def saved(name,
+          source='running',
+          user=None,
+          group=None,
+          mode=None,
+          attrs=None,
+          makedirs=False,
+          dir_mode=None,
+          replace=True,
+          backup='',
+          show_changes=True,
+          create=True,
+          tmp_dir='',
+          tmp_ext='',
+          encoding=None,
+          encoding_errors='strict',
+          allow_empty=False,
+          follow_symlinks=True,
+          check_cmd=None,
+          win_owner=None,
+          win_perms=None,
+          win_deny_perms=None,
+          win_inheritance=True,
+          win_perms_reset=False,
+          **kwargs):
+    '''
+    .. versionadded:: Fluorine
+
+    Save the configuration to a file on the local file system.
+
+    name
+        Absolute path to file where to save the configuration.
+        To push the files to the Master, use
+        :mod:`cp.push <salt.modules.cp.push>` Execution function.
+
+    source: ``running``
+        The configuration source. Choose from: ``running``, ``candidate``,
+        ``startup``. Default: ``running``.
+
+    user
```

Comment:
All of these have been copied from `states.file.managed`. I wasn't sure if I should just point to that function, but given that some arguments are not relevant in this context, e.g., `contents_pillar`, `contents_grains`, etc., I thought it would be better to just copy the documentation here to avoid any confusion.